common/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.quarkiverse.openapi.generator</groupId>
+    <artifactId>quarkus-openapi-generator-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+  <groupId>io.quarkiverse.xapi.generator</groupId>
+  <artifactId>quarkus-xapi-generator-common</artifactId>
+  <version>1.0.0</version>
```

Comment:
Since Im planning to use it as external dependency I think is should  have its own version

Reply:
On the other hand, if the repo does not support deploying with same version, this will certainly cause issues (a false failure), so, let use the same version than the parent. 
On the other hand this will create several versions of the library which will have the same content, which is bad. 
So we finally find the problem of doing cross dependency between repos;)

Reply:
As @ricardozanini sugggested offline, we migh consider adding a new library to KIE

Reply:
Moving to draft while I added the KIE library